hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/resources/log4j.properties

```diff
@@ -28,4 +28,11 @@ log4j.appender.CHAOS.File=${chaoslogfilename}
 log4j.appender.CHAOS=org.apache.log4j.FileAppender
 log4j.appender.CHAOS.layout=org.apache.log4j.PatternLayout
 log4j.appender.CHAOS.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c{2} (%F:%M(%L)) - %m%n
+
+log4j.appender.TEST.File=${errorlogfilename}
```

Comment:
Nit: I think we could find a better name to describe this appender, eg. `errors`, `problems` or something along that line.

Reply:
great suggestion, thanks